pkg/apis/apps/validation/validation_test.go

```diff
@@ -61,14 +61,14 @@ func TestValidateStatefulSet(t *testing.T) {
 		},
 	}
 
-	invalidTime := int64(60)
+	invalidTime := int64(0)
```

Comment:
isn't the purpose of this test case (error test case) to expect an error if the deadline is 0 seconds?
https://github.com/kubernetes/kubernetes/blob/b89aa6819f2e7999b09e706b823bc62079b59129/pkg/apis/apps/validation/validation_test.go#L344-L353